mappings/markdown/Zeek/v1.4.0/conn_log/README.md

```diff
@@ -0,0 +1,76 @@
+| `dst_endpoint.mac`            | `resp_l2_addr`  | Link-layer address of the responder, if available.                                      |                         |
+| `app_name`                    | `service`       | An identification of an application protocol being sent over the connection.            |                         |
+| `connection_info.community_uid` | `community_id`| The community ID.                                                                       |                         |
+| `connection_info.flag_history`  | `history`     | Records the state history of connections as a string of letters.                        |                         |
+| `connection_info.protocol_name` | `proto`       | The transport layer protocol of the connection.                                         |                         |
+| `duration`                    | `duration`      | How long the connection lasted.                                                         | Type is long_t.         |
+| `status_code`                 | `conn_state`    | Possible connection states.                                                             | Type is string.         |
+| `traffic.bytes_in`            | `resp_bytes`    | The number of payload bytes the responder sent.                                         | Type is long_t.         |
+| `traffic.packets_in`          | `resp_pkts`     | Number of packets that the responder sent.                                              | Type is long_t.         |
+| `traffic.bytes_out`           | `orig_bytes`    | The number of payload bytes the originator sent.                                        | Type is long_t.         |
+| `traffic.packets_out`         | `orig_pkts`     | Number of packets that the originator sent.                                             | Type is long_t.         |
+| `traffic.bytes_missed`        | `missed_bytes`  | Indicates the number of bytes missed in content gaps.                                   | Type is long_t.         |
+
+
+ ### Conditional mapping:
+Fields described here are subject to dynamic mappings contingent on a conditional evaluation of source data.
+| OCSF                          | Raw                              | Zeek Field Description                                                       | Evaluation Conditions                                                                                     |
+| ----------------------------- | -------------------------------- | ---------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| `activity_id`                 | `conn_state` | Possible connection states. | If "SF" or "RSTO" or "RSTR" or "RSTRH" or "SH" or "SHR" then "2" (Close) <br>if "S0" then "4" (Fail), <br>if "REJ" then "5" (Refuse), <br>if "OTH" or "S1" or "S2" or "S3" then "6" (Traffic) <br>else "6" (Traffic) <br>Type is Integer. |
```

Comment:
I took a deeper look at the `conn_state` mapping. Check out my analysis here: https://github.com/tenzir/library/pull/67/files